plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/provider/DockerExtConfBindingProvider.java

```diff
@@ -27,22 +33,38 @@
  */
 @Singleton
 public class DockerExtConfBindingProvider implements Provider<String> {
-    public static final  String EXT_CHE_LOCAL_CONF_DIR = "/mnt/che/conf";
-    private static final String CONTAINER_TARGET       = ":" + EXT_CHE_LOCAL_CONF_DIR + ":ro";
+
+    public static final String EXT_CHE_LOCAL_CONF_DIR = "/mnt/che/conf";
+
+    private static final String PLUGIN_CONF      = "plugin-conf";
```

Comment:
It makes sense to have the same names for folders with configuration of workspace agent. Your proposal to use ext and plugin-conf. Why?